### DISCUSSION TITLE
How to get OpenFDEM?

### DISCUSSION THREAD

**Discussion by krer-li** [OpenFDEM-geomechanics/OpenFDEM.github.io#3] [General]
I have sent an email to dr.Dr. Xiaofeng Li, but there is no reply, I wonder if there is any condition to use the software, please tell me!

**Comment by pfefailed** [comment]
We did the same thing, but there is no reply.

**Comment by krer-li** [comment]


**Reply by pfefailed** [reply]
Yes we urgently need the code.

**Reply by masterjoker12391** [reply]
We need it，too. Could you tell me?

**Comment by HongningQi** [comment]
How to install on the Windows system? I need a tutorial.
